openapi/components/schemas/Websites/WebsiteId.yaml

```
@@ -0,0 +1,7 @@
+type: string
+description: |-
+  Unique website identifier string.
+  A website is where an organization obtains a customer through a subscription.
+  For more information, see [Obtain an organization ID and website ID](https://www.rebilly.com/docs/settings/organizations-and-websites/#obtain-your-organization-id-and-website-id).
+maxLength: 50
+example: 4f6cf35x-2c4y-483z-a0a9-158621f77a21
```

```suggestion
example: example.com
```
I'm making this identifier different than the others, or they all blend together.